README.md

```diff
@@ -65,12 +65,21 @@ In order to use these scripts, you will need to set a few things up.
 - [Python Click](https://github.com/mitsuhiko/click) version 4.0 or greater
 - [Python Boto](http://docs.pythonboto.org) version 2.38.0 or greater
 - [Ansible](https://github.com/ansible/ansible) version 1.9.4
+- pyopenssl version 0.15.1
 
 Python and the Python dependencies may be installed via your OS' package manager
 (eg: python-click on Fedora/CentOS/RHEL) or via
 [pip](https://pypi.python.org/pypi/pip). [Python
 virtualenv](https://pypi.python.org/pypi/virtualenv) can also work.
 
+Example of installation of packaged software
```

Comment:
Makes an assumption that you are going to use pip. If you're going to document pip then you need to document every other packaging mechanism, which theoretically includes non-RPM distributions, python virtualenv, etc etc.

I think these instructions are out of scope.